easy-ui-react/src/InputField/InputField.module.scss

```
@@ -162,3 +162,39 @@
 .caption {
   margin-top: design-token("space.0.5");
 }
+
+.textStartInput {
+  @include Input.textStartInput;
+}
+
+.textEndInput {
+  @include Input.textEndInput;
+}
+
+.inputTextPlacementSm {
+  top: design-token("space.1");
+}
+
+.inputTextPlacementLg {
+  top: design-token("space.2");
+}
+
+.text {
+  border: 0;
+  display: inline-block;
+  position: absolute;
+  top: design-token("space.1.5");
+  z-index: component-token("inputfield", "z-index");
+  min-width: 24px;
+  height: 24px;
+}
+
+.textStart {
+  left: design-token("space.1.5");
+  color: theme-token("color.gray.600");
```

prefer to use `neutral` instead of `gray` and the `theme-token()` helper is now deprecated:

```suggestion
  color: design-token("color.neutral.600");
```